src/components/learner-credit-management/data/hooks/useOfferRedemptions.js

```diff
@@ -69,14 +78,27 @@ const useOfferRedemptions = (enterpriseUUID, offerId = null, budgetId = null) =>
           applySortByToOptions(args.sortBy, options);
         }
         if (args.filters?.length > 0) {
-          applyFiltersToOptions(args.filters, options);
+          applyFiltersToOptions(args.filters, options, fetchTransactions);
         }
-        const response = await EnterpriseDataApiService.fetchCourseEnrollments(
-          enterpriseUUID,
-          options,
-        );
-        const data = camelCaseObject(response.data);
-        const transformedTableResults = transformUtilizationTableResults(data.results);
+        let data;
+        let transformedTableResults;
+        if (offerId === null && fetchTransactions) { // and do the feature flag check here
```

Comment:
[curious] Just wondering on this check against `offerId` being null versus having a non-null `budgetId` (subsidy access policy uuid). If this conditional is to determine whether we're dealing with a subsidy access policy, it might make sense to rely on the `budgetId` variable instead.

Reply:
¯\_(ツ)_/¯ I had no particular method to the madness, just what came out first. Makes sense to have it the other way